src/Service/CommandHelper.php

```diff
@@ -16,11 +16,14 @@ final class CommandHelper
 
     private ?string $consoleBin = null;
 
+    private ?float $timeout = null;
```

Comment:
```suggestion
    private ?float $timeout;
```
Default value is not necessary because the value is set in constructor.

- [ ] No default value for property